CHANGELOG.md

```diff
@@ -58,6 +58,10 @@ Unreleased changes template.
 ### Fixed
 * (py_wheel) Use the default shell environment when building wheels to allow
   toolchains that search PATH to be used for the wheel builder tool.
+* (pip.parse) The requirement argument parsed to `whl_library` will now not have
+  env marker information allowing `bazel query` to work in cases where the `whl`
+  is available for all of the platforms and the sdist can be built.
+  Work towards [#2450](https://github.com/bazelbuild/rules_python/issues/2450).
```

Comment:
```suggestion
  Fixes [#2450](https://github.com/bazelbuild/rules_python/issues/2450).
```

Actually, @keith, I think that this PR might fix the issue you were having. Let me know if it does in something else than #2450 as with this PR the command `bazel query deps(...)` no longer fails.

Reply:
looks like it does! 🎉